freedomofpress/securedrop
# Test tor packages from `tor-apt-test` in release branches

## Status

Work in progress

## Description of Changes

Partially fixes  #2919  -- performs testing against CI staging AFTER installing latest tor package in `tor-apt-test.freedom.press`

Changes proposed in this pull request:

Ummm what I described right above this part... GEEEZ!!

## Testing

How should the reviewer test this PR?

You want to check to make sure CI passed and confirm you are able to see a `.tor_version` artifact in circle CI and it's content matches version `0.3.2.9-1~trusty+1`


## Deployment

Any special considerations for deployment? Consider both:

Well this PR affects CI specfically, but locally for QA you also want to make sure you are installing tor from tor-test-apt.freedom.press

## Checklist

### If you made changes to the system configuration:

- [] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

Runing tests locally now, I had to make a tweak to the tor apt test so it doesnt run in the release* branches (same behavior as when this tor apt test is supposed to run).
